Src/java/elm-jackson/src/main/java/org/cqframework/cql/elm/serializing/jackson/ElmLibraryReaderProvider.java

```diff
@@ -0,0 +1,18 @@
+package org.cqframework.cql.elm.serializing.jackson;
+
+import org.cqframework.cql.elm.serializing.ElmLibraryReader;
+
+public class ElmLibraryReaderProvider implements org.cqframework.cql.elm.serializing.ElmLibraryReaderProvider {
+    @Override
+    public ElmLibraryReader create(String contentType) {
+        if (contentType == null) {
+            contentType = "application/elm+json";
+        }
+
+        switch (contentType) {
+            case "application/elm+xml": return new ElmXmlLibraryReader();
```

Comment:
We should probably add an issue to create add a `text/cql` reader that compiles the code. Would it make sense?

Reply:
Should this `contentType` also check for different CQL versions and send it to different readers?

Reply:
I think maybe keep compilation and serialization separate? We have facilities for this in the loader layer in the evaluator and ruler, so I don't think we need to introduce that here

Reply:
The versions question is a good one, but we should maybe deal with that separately. At this point all ELM is backwards compatible, so it shouldn't be an issue with using the latest readers